code/modules/projectiles/guns/energy/special.dm

```diff
@@ -159,6 +159,16 @@
 	else
 		..()
 
+/obj/item/gun/energy/plasmacutter/emp_act(severity)
+	if(!cell.charge)
+		return
+	cell.use(cell.charge/3)
+	if(isliving(loc))
+		var/mob/living/L = loc
+		L.visible_message("<span class='danger'>Searing plasma is spewed out of [L]'s [src]!</span>", "<span class='userdanger'>Your [src] malfunctions and spews incinerating plasma onto you!</span>")

```

Comment:
"Incinerating plasma" is a bad phrase imo

Reply:
Yeah I am god dam horrid with words to be honest

Reply:
Also "searing plasma spews from" is better phrasing here.

Reply:
```suggestion
		L.visible_message("<span class='danger'>Concentrated plasma discharges from [src] onto [L], burning them!</span>", "<span class='userdanger'>Your [src] malfunctions, spewing concentrated plasma onto you! It burns!</span>")
```